exchanges/huobi/huobi_wrapper.go

```diff
@@ -946,9 +946,9 @@ func (h *HUOBI) SubmitOrder(ctx context.Context, s *order.Submit) (order.SubmitR
 		var oType string
 		switch s.Type {
 		case order.Market:
-			oType = "opponent"
+			oType = "optimal_20"
 			if s.ImmediateOrCancel {
-				oType = "opponent_ioc"
+				oType = "optimal_20_ioc"
```

Comment:
Going from docs: 

> At present, Huobi Futures does not support market price when placing an order. To increase the probability of a transaction, users can choose to place an order based on BBO price (opponent), optimal 5 (optimal_5), optimal 10 (optimal_10), optimal 20 (optimal_20), among which the success probability of optimal 20 is the largest, while the slippage always is the largest as well.
> 
> It is important to note that the above methods will not guarantee the order to be filled in 100%. The system will obtain the optimal N price at that moment and place the order.

So I guess maybe a note stating the fact that if the amount to deploy to and from the ask/bid liquidity exceeds N price levels, it will not be filled and further requests might need to be undertaken?

Reply:
sounds good, i'm linking the Huobi doc and pasted this snippet which makes things clearer